R/helpers_covr.R

```diff
@@ -0,0 +1,54 @@
+#' Determine whether `covr` is currently running
+#'
+h_covr_active <- function() {
+  identical(Sys.getenv("R_COVR"), "true")
+}
+
+#' Remove `covr` traces from an expression
+#'
+#' '`covr` interferes with expressions (often function bodies) as a primary
+#' interface of this package. To work with `covr`, we need to strip out the
+#' code that `covr` injects into the package before using these expressions
+#' for internal logic such as writing model files.
+#'
+#' This method is non-exhaustive, covering only a subset of `covr`'s tracing
+#' behaviors necessary for this package.
+#'
+#' @param expr (`language`)\cr an R expression or call to strip of `covr` trace
```

Comment:
that reminds me we still should transform this package to using `roxytypes` and `roxylint`

Reply:
Going to mark this one as resolved, but this could certainly be a separate cleanup task. I'm curious to hear how they're working out for `savvyr` or any other repos you've adopted them for.